### PR TITLE
New website

### PR DESCRIPTION
**Pull request by luise**

### CODE REVIEW

File: kelda/deploy.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const kelda = require('kelda');
+
+const vm = new kelda.Machine({ provider: 'Amazon' });
+const infrastructure = new kelda.Infrastructure({ masters: vm, workers: vm });
+
+const website = new kelda.Container({ name: 'website', image: 'luise/kelda.io:0.1' });
```
Comment (kklin) [inline kelda/deploy.js]: I think the image should be hosted under the `keldaio` user. I'll give you push access.

File: kelda/package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@kelda/kelda.io",
+  "version": "0.0.1",
+  "main": "./deploy.js",
+  "license": "MIT",
+  "dependencies": {
+    "kelda": ">=0.0.5 <1.0.0"
+  },
+  "devDependencies": {
+    "eslint": "^4.4.1",
```
Comment (kklin) [inline kelda/package.json]: It doesn't look like you actually use these dependencies

**Comment by luise** [comment]
Oh, wow, thanks for the quick review! The blueprint is just for development, since I can't think of a good way to reuse it in the infra repo. I was going to write a separate version for the infra blueprint - ideally, the website repo will auto build an image, and we'll update the website by updating the tag in the blueprint (rather than restarting and thus rebuilding the image in the cluster as we do now).

File: kelda/package.json (hunk 2 above)
Comment (luise) [reply kelda/package.json]: Good point!

File: kelda/deploy.js (hunk 1 above)
Comment (luise) [reply kelda/deploy.js]: Thanks!

File: site/data/landing.yaml [hunk 3]
```diff
@@ -2,42 +2,41 @@
...
   headline: "Soar through your <em class='tooltip' title='Containers are a combination of several kernel features including SELinux, Linux namespaces, and control groups, providing isolation of end user processes, networking, and filesystem space.'>Containerized Application</em> configurations with Kelda."
   subhead: "Simply describe with Javascript and deploy from the command line."
-  npmcode: "npm install -g @install/kelda"
+  npmcode: "npm install -g @kelda/install"
   btnsubtext: "Got questions? [Take a tour of Kelda →](http://docs.kelda.io/#quick-start)"
   examplecode: |
     ```js
-      // Boot virtual machines in seconds
+      // Boot virtual machines.
       const vm = new Machine({
```
[the remark this answers is not in the document]
Comment (luise) [reply site/data/landing.yaml]: I'm not quite sure what I prefer. This is valid if you just imported it with the `{ Machine, ... }` syntax. Do you have a preference?

**Comment by kklin** [comment]
Ooh I see. If it's just for development I don't think the image needs to be pushed under the keldaio user.

**Pull request merged**